File: 2018/10/hello/fizzbuzz.cc
#include <iostream>

int main()
{
    for (int i = 0; i < 100; i++)
    {
        if (i % 15 == 0)
        {
            std::cout << "fizz: " << i << std::endl;
        }
        else if (i % 5 == 0 || i % 3 == 0)
        {
            std::cout << "buzz: " << i << std::endl;
        }
        else
        {
            std::cout << "fizzbuzz: " << i << std::endl;
        }
    }
}

File: 2018/11/fizzbuzz/fizzbuzz.py
import sys

for line in sys.stdin:
    X, Y, N = map(int, line.split(' '))

for i in range(1, N + 1):
    if ((i % X) + (i % Y)) == 0:
        print('FizzBuzz')
    elif i % X == 0:
        print('Fizz')
    elif i % Y == 0:
        print('Buzz')
    else:
        print(i)
